src/helm/config/model_metadata.yaml

```diff
@@ -1394,7 +1394,15 @@ models:
     release_date: 2023-10-05
     tags: [TEXT_MODEL_TAG, LIMITED_FUNCTIONALITY_TEXT_MODEL_TAG]
 
-
+  # KAIST AI
+  - name: kaistai/prometheus-vision-13b-v1.0-hf
```

Comment:
Is this there best model?

Reply:
Yes, the 13B-v1.0 model is their best VL evaluator now